Bin/Snapshot.php

```diff
@@ -49,19 +49,34 @@
  */
 class Snapshot extends Console\Dispatcher\Kit
 {
+    /**
+     * User friendly labels for changelog themes.
+     *
+     * @const
+     */
+    public const CHANGELOG_THEMES = [
+        'chore' => 'Quality',
+        'fix'   => 'Bug fixes',
+        'feat'  => 'New features',
+        'doc'   => 'Documentation modification',
+        'test'  => 'Test modification',
+        'undef' => 'Anything else'
+    ];
+
     /**
      * Options description.
      *
      * @var array
      */
     protected $options = [
-        ['only-changelog',      Console\GetOption::NO_ARGUMENT,       'c'],
-        ['only-tag',            Console\GetOption::NO_ARGUMENT,       't'],
-        ['only-github-release', Console\GetOption::NO_ARGUMENT,       'g'],
-        ['break-bc',            Console\GetOption::NO_ARGUMENT,       'b'],
-        ['minimum-tag',         Console\GetOption::REQUIRED_ARGUMENT, 'm'],
-        ['help',                Console\GetOption::NO_ARGUMENT,       'h'],
-        ['help',                Console\GetOption::NO_ARGUMENT,       '?']
+        ['only-changelog',          Console\GetOption::NO_ARGUMENT,       'c'],
+        ['only-structured-commits', Console\GetOption::NO_ARGUMENT,       's'],
```

Comment:
Intuitively, I would have used “typed” instead of “structured”. What do you think?

Reply:
And why would we want to skip untyped commits? It's an opened question. I've no opinion yet.

Reply:
I've added that option after reading the keepachangelog.com guidelines another time. Adding all the commits in the changelog is not always very interesting because some haven't any meaning for the updates.

This option let the choice to retrieve them or not...

Reply:
Hmm, why not, I'm good to keep it if you think it is useful.